.github/workflows/go-build.yml

```diff
@@ -18,13 +18,9 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9
       with:
-        go-version: 1.18
+        go-version: 1.19
 
     - name: Build
-      env:
-        CGO_ENABLED: 0
-        GOOS: linux      
-      run: go build --tags static_all -v -o ./bin/netpolicy cmd/netpolicy/main.go
-    
+      run: make build
     - name: Test
-      run: go test -v ./...
+      run: make test
```

Comment:
why do we need these changes?

Reply:
they are not necessary (I just was trying to figure if the build failure was connected to the commands or not).
however I think we better use these , this way we avoid duplication by writing the build and test commands only once (in the make file) and use them for both local and on server runs.

This way (using make) is used in cluster-topology-analyzer's workflow too